artifact-lang.sh

```diff
@@ -3,14 +3,14 @@ set -eu
 
 function retrieveLanguageParameters() {
   declare -a languages
-  languages=(en fr de nl)
+  languages=(de en es fr it nl)
```

Comment:
can we remove this dependency and get the list from the json file as below
```suggestion
  languages=(`jq --raw-output '.resources | keys | @tsv' locales/languages.json`)
```
if we use jq, we have to be sure it's installed when running in jenkins

Reply:
Or without jq:
```bash
echo 'const l=require("./locales/languages.json"); console.log(Object.getOwnPropertyNames(l.resources).join(" "));' | node -
```

Reply:
no more dependency, great !!!